bin/resources/GameIndex.yaml

```diff
@@ -140,6 +140,11 @@ PCPX-96649:
 PCPX-96660:
   name: "Tourist Trophy [Demo]"
   region: "NTSC-J"
+PCPX-98017:
+  name: "Ratchet & Clank 4th - GiriGiri Ginga no Giga Battle [Demo]"
+  region: "NTSC-J"
+  gameFixes:
+    - VUKickstartHack # Fixes Character SPS.
```

Comment:
Do not forget the Timing hack, it is best to have it on on those games.

Reply:
Thanks :)

Reply:
@gilderoylockhart01  taken care of :+1: